org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/LanguageServerImpl.xtend

```diff
@@ -194,6 +196,12 @@ import static org.eclipse.xtext.diagnostics.Severity.*
 		], []).thenApply [result]
 	}
 	
+	override void initialized(InitializedParams params) {
+		requestManager.runWrite([
```

Comment:
why is the runWrite needed here?

Reply:
now directly calling` initialized.complete()`